cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -893,6 +913,22 @@ func getEnvWithDefaultEmptySet(k string) map[string]struct{} {
 	return set
 }
 
+// findName returns the name of the current process, that being the
+// part of argv[0] after the last slash if any, and also the lowercase
+// letters from that, suitable for use as a likely key for lookups
+// in things like shell environment variables which can't contain
+// hyphens.
+// Copied from https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/env/env.go?L55:6
+func findName() (string, string) {
+	// Environment variable names can't contain, for instance, hyphens.
+	origName := filepath.Base(os.Args[0])
+	name := strings.ReplaceAll(origName, "-", "_")
+	if name == "" {
+		name = "unknown"
+	}
+	return origName, name
+}
```

Comment:
we don't need this, we know the binary name since this is a main pacakge `zoekt-sourcegraph-indexserver`

Reply:
removed